src/styles/components/_home-search.scss

```diff
@@ -75,4 +76,18 @@
       font-size: rem(26px);
     }
   }
+
+  &__box {
+    @include flex-center();
+    position: absolute;
+    width: 100%;
+
+    top: 0%;
+    right: 0%;
+    left: 0%;
```

Comment:
Currently it's not perfectly center, this might help.
Here: https://css-tricks.com/quick-css-trick-how-to-center-an-object-exactly-in-the-center/

Reply:
It's still not perfectly centered, try the last option on the link I provided first.

Reply:
Updated: I applied your suggestion and it works great. For `medium size`, `small size`, and below, I'll apply like this:
```
&__box {
    @include flex-center();

    position: absolute;
    width: 100%;
    top: 0%;

    @include respond-above('md') {
      top: 50%;
      left: 50%;
      transform: translate(-50%, -50%);
    }
  }
```
What do you think?

Since the box is in the top position with a horizontal center from full 100% width. 
The padding is pushed to make space for the search box.

Reply:
Sure, same thought as you.